src/content/en/updates/2020/05/devtools.md

```diff
@@ -0,0 +1,126 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: TODO
+
+{# wf_updated_on: 2020-05-11 #}
+{# wf_published_on: 2020-05-11 #}
+{# wf_tags: chrome84, devtools, devtools-whatsnew #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: TODO #}
+{# wf_blink_components: Platform>DevTools #}
+
+# What's New In DevTools (Chrome 84) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+## Issues panel {: #issues }
+
+The new **Issues** panel was built to help reduce the notifcation fatigue and
+clutter of the **Console**. Currently, the Console is the central place for
+website developers, libraries, frameworks, and Chrome itself to log messages,
+warnings, and errors. The Issues panel structures and aggregates all of these
```

Comment:
As it stands, the Issues tab only aggregates messages from the browser: there's no intention at present to provide an API to make it accessible from JavaScript (i.e. for developers or frameworks, library or platforms). 

@danieltxok can confirm.

Maybe something like this: 'The Issues tab aggregates warnings from the browser in a structured format, linking to affected resources within DevTools, and providing guidance on how to fix problems.'

Reply:
Correct, currently only showing browser messages. Adding small changes to Sam's proposal:
'The Issues tab aggregates warnings from the browser in a structured, aggregated and actionable way, linking to affected resources within DevTools, and providing guidance on how to fix the issues.